python/pyspark/sql/readwriter.py

```diff
@@ -120,6 +120,9 @@ def option(self, key, value):
             * ``timeZone``: sets the string that indicates a timezone to be used to parse timestamps
                 in the JSON/CSV datasources or partition values.
                 If it isn't set, it uses the default value, session local timezone.
+            * ``pathGlobFilter``: an optional glob pattern to only include files with paths matching
+                the pattern. The syntax follows org.apache.hadoop.fs.GlobFilter.
+                It does not change the behavior of partition discovery.
```

Comment:
nit: I would mention it's "For file based sources such as Parquet and ORC, blah blah ..."

Reply:
I think the current one is OK,  as the option is explained with examples in document.